src/doc/rustc/src/platform-support/hexagon-unknown-none-elf.md

```diff
@@ -41,6 +41,8 @@ target = ["<target for your host>", "hexagon-unknown-none-elf"]
 cc = "hexagon-unknown-none-elf-clang"
 cxx = "hexagon-unknown-none-elf-clang++"
 linker = "hexagon-unknown-none-elf-clang"
+ranlib = "hexagon-unknown-linux-musl-ranlib"
+ar = "hexagon-unknown-linux-musl-ar"
```

Comment:
Shouldn't this be using the `none-elf` toolchain instead of the musl one?

Reply:
Indeed, thanks for catching this.  In practice I guess it wouldn't have mattered functionally.  But it was wrong and now fixed.